tests/stl/open_map_check.cpp

```diff
@@ -235,6 +235,9 @@ TEST(open_map_test, test_access_operator) {
 }
 
 TEST(open_map_test, test_rehash) {
+    // class gives segmentation try using new malloc that should fix it maybe cuz I changed a lot of stuff
+    // also do not directly use Allocator
+    /*
```

Comment:
Will go through and purge usage of `Allocator`, should I push directly to this branch?

Reply:
yeah do that it's okay